src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/responsibilitiesRepository/GetResponsibilityTests.kt

```diff
@@ -149,23 +149,24 @@ class GetResponsibilityTests : ResponsibilitiesRepositoryTests()
         }
     }
 
-    @org.junit.Test
+    @Test
     fun `can get responsibility`()
     {
         given {
             it.addGroup("group-1", "description")
             it.addTouchstoneVersion("touchstone", 1, "description", "open", addTouchstone = true)
             it.addScenarioDescription("scenario-1", "description", "disease", addDisease = true)
             val setId = it.addResponsibilitySet("group-1", "touchstone-1", "incomplete")
-            it.addResponsibility(setId, "touchstone-1", "scenario-1")
+            val responsibilityId = it.addResponsibility(setId, "touchstone-1", "scenario-1")
+            it.addExpectations(responsibilityId)
```

Comment:
Is this neccessary? This method should work without expectations right?

Reply:
Correct. I changed that test and then changed it back, but clearly left some bits and pieces lying around. I'll fix it